CODEOWNERS

```diff
@@ -0,0 +1,6 @@
+.drone.yml @oxr463 @paraglade @rosskirkpat
```

Comment:
I don't think this should be EIO.

Reply:
I agree that it should not be wholly owned by EIO, however, at this time I don't think any Engineering team (or sub-set of a team, like what EIO is), has a clear ownership/knowledge of Drone other than EIO. That was why I included both of you. This is something we should address at a higher level of SMEs and technical ownership of a product/project.

I know that @phillipsj and I have spent a lot of time working in Drone across many projects in Rancher. I will change this to include the two of us and @superseb